app/src/main/java/com/nextcloud/talk/controllers/CallController.java

```diff
@@ -2360,6 +2360,15 @@ private void setCallState(CallStatus callState) {
         }
     }
 
+    private String getDescriptionForCallType() {
+        String appName = Objects.requireNonNull(getResources()).getString(R.string.nc_app_name);
```

Comment:
If you think this might be null, then you also need to handle resulting NPE.

Reply:
ah forgot to change this. actually we don't check anywhere for NPE for resources. so i stick with this. or any concerns?